pyproject.toml

```diff
@@ -79,7 +79,8 @@ show_error_codes = "True"
 files = [
   "tuf/api/",
   "tuf/ngclient",
-  "tuf/exceptions.py"
+  "tuf/exceptions.py",
+  "examples/",
 ]
```

Comment:
left a comment about this in the other bug: https://github.com/theupdateframework/python-tuf/pull/1699#discussion_r760934677 (we could remove these and just define the "linter dirs" in one place in tox.ini to make maintenance easier)